test/cassettes/test_determine_and_release_claim.yaml

```diff
@@ -0,0 +1,216 @@
+interactions:
+- request:
+    body: null
+    headers:
+      Accept:
+      - application/vnd.v1+json
+      Accept-Encoding:
+      - gzip, deflate
+      Authorization:
+      - Bearer eyJhbGciOiJIUzUxMiIsInR5cCI6IkpXVCJ9.eyJhdWQiOiJUdXJuIiwiZXhwIjoxNzc4MDAxNjYwLCJpYXQiOjE3MTQ5MzMyNzEsImlzcyI6IlR1cm4iLCJqdGkiOiIzYWNjY2VmYS00ZWFkLTQ0MzctYWNhMi04N2IyOWU3Yjk0OTYiLCJuYmYiOjE3MTQ5MzMyNzAsInN1YiI6Im51bWJlcjozOTc3IiwidHlwIjoiYWNjZXNzIn0.I-qhamvyd3FH_2kJlioHI_jmg4NVROAujNQCYJIVWct1FXWXyVMJoNsyTcLJGhCpQeMaNJpmRVwkgMu7KDFXZg
```

Comment:
Hmmm, good catch. I have now expired this token, so these are now safe. Actually you cannot run the test suite anyway with these cassettes because the values passed if anyone else tries to run them will be different, so the cassettes will re-record. I think it is safe to remove these from the repo completely (they will obviously still be in the commit history though). I will add the necessary information to the README.